src/formats/bear-bear2bk.ts

```diff
@@ -50,7 +56,13 @@ export class Bear2bkImporter extends FormatImporter {
 								mdContent = mdContent.replace(assetMatcher, `![](${attachmentsFolderPath.path}/`);
 							}
 							let filePath = normalizePath(mdFilename);
-							await this.saveAsMarkdownFile(outputFolder, filePath, mdContent);
+							const file = await this.saveAsMarkdownFile(outputFolder, filePath, mdContent);
+
+							const writeOptions: DataWriteOptions = {
+								ctime: metaData[parent].ctime,
+								mtime: metaData[parent].mtime,
+							};
+							await this.vault.append(file, '', writeOptions);
```

Comment:
Yeah I don't think there's a better way for now given the APIs exposed.

Reply:
Sounds good, thanks.

Reply:
I think I copied that approach from the `onenote.ts` format. 

I just checked the `notion.ts` format, there it is a single call `vault.create(path, markdownBody, writeOptions)` instead of `this.saveAsMarkdownFile(outputFolder, filePath, mdContent)`. I don't know what's the difference? I can give it a try!